{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "children": [],
        "cpe_match": [
          {
            "cpe23Uri": "cpe:2.3:o:linux:linux_kernel:*:*:*:*:*:*:*:*",
            "cpe_name": [],
            "versionEndExcluding": "5.18.4",
            "versionStartIncluding": "5.18",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:o:linux:linux_kernel:*:*:*:*:*:*:*:*",
            "cpe_name": [],
            "versionEndExcluding": "5.17.15",
            "versionStartIncluding": "5.16",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:o:linux:linux_kernel:*:*:*:*:*:*:*:*",
            "cpe_name": [],
            "versionEndExcluding": "5.15.47",
            "versionStartIncluding": "5.11",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:o:linux:linux_kernel:*:*:*:*:*:*:*:*",
            "cpe_name": [],
            "versionEndExcluding": "5.10.122",
            "versionStartIncluding": "5.5",
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:o:linux:linux_kernel:*:*:*:*:*:*:*:*",
            "cpe_name": [],
            "versionEndExcluding": "5.4.198",
            "versionStartIncluding": "3.9",
            "vulnerable": true
          }
        ],
        "operator": "OR"
      }
    ]
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@kernel.org",
      "ID": "CVE-2022-49385"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "In the Linux kernel, the following vulnerability has been resolved:\n\ndriver: base: fix UAF when driver_attach failed\n\nWhen driver_attach(drv); failed, the driver_private will be freed.\nBut it has been added to the bus, which caused a UAF.\n\nTo fix it, we need to delete it from the bus when failed."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://git.kernel.org/stable/c/310862e574001a97ad02272bac0fd13f75f42a27",
          "refsource": "",
          "tags": [
            "Patch"
          ],
          "url": "https://git.kernel.org/stable/c/310862e574001a97ad02272bac0fd13f75f42a27"
        },
        {
          "name": "https://git.kernel.org/stable/c/5389101257828d1913d713d9a40acbe14f5961df",
          "refsource": "",
          "tags": [
            "Patch"
          ],
          "url": "https://git.kernel.org/stable/c/5389101257828d1913d713d9a40acbe14f5961df"
        },
        {
          "name": "https://git.kernel.org/stable/c/5d709f58c743166fe1c6914b9de0ae8868600d9b",
          "refsource": "",
          "tags": [
            "Patch"
          ],
          "url": "https://git.kernel.org/stable/c/5d709f58c743166fe1c6914b9de0ae8868600d9b"
        },
        {
          "name": "https://git.kernel.org/stable/c/823f24f2e329babd0330200d0b74882516fe57f4",
          "refsource": "",
          "tags": [
            "Patch"
          ],
          "url": "https://git.kernel.org/stable/c/823f24f2e329babd0330200d0b74882516fe57f4"
        },
        {
          "name": "https://git.kernel.org/stable/c/c059665c84feab46b7173d3a1bf36c2fb7f9df86",
          "refsource": "",
          "tags": [
            "Patch"
          ],
          "url": "https://git.kernel.org/stable/c/c059665c84feab46b7173d3a1bf36c2fb7f9df86"
        },
        {
          "name": "https://git.kernel.org/stable/c/cdf1a683a01583bca4b618dd16223cbd6e462e21",
          "refsource": "",
          "tags": [
            "Patch"
          ],
          "url": "https://git.kernel.org/stable/c/cdf1a683a01583bca4b618dd16223cbd6e462e21"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2025-03-25T14:58Z",
  "publishedDate": "2025-02-26T07:01Z"
}